frontend/src/e2e-playwright/specs/0_citizen/citizen-income-statement.spec.ts

```diff
@@ -29,13 +29,26 @@ afterEach(async () => {
   await page.close()
 })
 
-async function assertIncomeStatementCreated(startDate: string) {
+async function assertIncomeStatementCreated(
+  startDate = LocalDate.today().format()
+) {
   await waitUntilEqual(async () => await incomeStatementsPage.rows.count(), 1)
   await waitUntilTrue(async () =>
     (await incomeStatementsPage.rows.innerText()).includes(startDate)
   )
 }
 
+async function assertNeededAttachment(attachment: string, present = true) {
+  await waitUntilTrue(
+    async () =>
+      (!present &&
+        !(await incomeStatementsPage.requiredAttachments.isVisible())) ||
+      (
+        await incomeStatementsPage.requiredAttachments.innerText()
+      ).includes(attachment) === present
+  )
```

Comment:
```suggestion
  await waitUntilTrue(async () => 
    present
      ? (await incomeStatementsPage.requiredAttachments.innerText()).includes(attachment)
      : !(await incomeStatementsPage.requiredAttachments.isVisible())
  )
```

Reply:
The proposed change would change the semantics.

if not present: either `the list is not visible at all` or `the element is not present in the list`
if present: `the element is present in the list`